src/main/java/net/unaussprechlich/warlordsplus/ingamegui/components/scoreboard/ScoreboardComponent.kt

```diff
@@ -64,111 +48,109 @@ object ScoreboardComponent : AbstractRenderComponent() {
         val xReceived = w2 + spacing * 3
 
 
-        drawRect(xStart, yStart, w, 13, Color(34, 34, 39, 200))
-        drawString(xLevel, yStart + 3, "Class Lvl  Name", false)
+        drawHeaderRect(xStart, yStart, w, 13)
+        drawString(xLevel, yStart + 3, "Class Lvl   Name", false)
         drawString(xKills - spacing / 5, yStart + 3, "Kills", false)
         drawString(xDeaths - spacing / 4, yStart + 3, "Deaths", false)
         drawString(xDone - spacing / 5, yStart + 3, "Given", false)
         drawString(xReceived - spacing / 3, yStart + 3, "Received", false)
 
         var offset = 0
 
-
-
-        drawRect(xStart, yStart + 15, w, 12 * teamBlue.size, Color(34, 34, 39, 100))
+        drawBackgroundRect(xStart, yStart + 14, w, 10 * teamBlue.size)
         for (p in teamBlue) {
```

Comment:
Requires some cleanup. It is almost the same code for Blue and Red

Reply:
supposed to be?

Reply:
Clean up the entire method, can be done in half the amount of lines, so we can add stuff easier later on